[Fabric] server crash on startup

HQM-1.16.4+5.5.8-fabric
fabric-api-0.35.0+1.16
```
java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'hardcorequesting'!
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:53) ~[fabric-server-launch.jar:?]
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke(EntrypointUtils.java:36) ~[fabric-server-launch.jar:?]
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointServer.start(EntrypointServer.java:32) ~[fabric-server-launch.jar:?]
	at net.minecraft.server.Main.main(Main.java:92) [intermediary-server.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_261]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_261]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_261]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_261]
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226) [fabric-server-launch.jar:?]
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:146) [fabric-server-launch.jar:?]
	at net.fabricmc.loader.launch.knot.KnotServer.main(KnotServer.java:28) [fabric-server-launch.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_261]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_261]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_261]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_261]
	at net.fabricmc.loader.launch.server.FabricServerLauncher.launch(FabricServerLauncher.java:62) [fabric-server-launch.jar:?]
	at net.fabricmc.loader.launch.server.FabricServerLauncher.setup(FabricServerLauncher.java:106) [fabric-server-launch.jar:?]
	at net.fabricmc.loader.launch.server.FabricServerLauncher.main(FabricServerLauncher.java:50) [fabric-server-launch.jar:?]
Caused by: java.lang.NoClassDefFoundError: net/minecraft/class_638
	at hardcorequesting.common.fabric.proxies.CommonProxy.init(CommonProxy.java:12) ~[HQM-1.16.4+5.5.8-fabric.jar:?]
	at hardcorequesting.common.fabric.HardcoreQuestingCore.initialize(HardcoreQuestingCore.java:61) ~[HQM-1.16.4+5.5.8-fabric.jar:?]
	at hardcorequesting.fabric.HardcoreQuestingFabric.onInitialize(HardcoreQuestingFabric.java:83) ~[HQM-1.16.4+5.5.8-fabric.jar:?]
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:50) ~[fabric-server-launch.jar:?]
	... 17 more
Caused by: java.lang.ClassNotFoundException: net.minecraft.class_638
	at java.net.URLClassLoader.findClass(Unknown Source) ~[?:1.8.0_261]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.8.0_261]
	at sun.misc.Launcher$AppClassLoader.loadClass(Unknown Source) ~[?:1.8.0_261]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.8.0_261]
	at net.fabricmc.loader.launch.server.InjectingURLClassLoader.loadClass(InjectingURLClassLoader.java:56) ~[fabric-server-launch.jar:?]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.8.0_261]
	at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:168) ~[fabric-server-launch.jar:?]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.8.0_261]
	at hardcorequesting.common.fabric.proxies.CommonProxy.init(CommonProxy.java:12) ~[HQM-1.16.4+5.5.8-fabric.jar:?]
	at hardcorequesting.common.fabric.HardcoreQuestingCore.initialize(HardcoreQuestingCore.java:61) ~[HQM-1.16.4+5.5.8-fabric.jar:?]
	at hardcorequesting.fabric.HardcoreQuestingFabric.onInitialize(HardcoreQuestingFabric.java:83) ~[HQM-1.16.4+5.5.8-fabric.jar:?]
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:50) ~[fabric-server-launch.jar:?]
	... 17 more
```

--- a/common/src/main/java/hardcorequesting/common/proxies/ClientProxy.java
+++ b/common/src/main/java/hardcorequesting/common/proxies/ClientProxy.java
@@ -1,5 +1,6 @@
 package hardcorequesting.common.proxies;
 
+import hardcorequesting.common.HardcoreQuestingCore;
 import hardcorequesting.common.client.sounds.Sounds;
 import hardcorequesting.common.network.PacketContext;
 import hardcorequesting.common.quests.Quest;
@@ -16,7 +17,8 @@ public class ClientProxy extends CommonProxy {
     @Override
     public void init() {
         super.init();
-        Quest.clientTicker = QuestTicker.initClientTicker();
+        Quest.clientTicker = new QuestTicker();
+        HardcoreQuestingCore.platform.registerOnClientTick(minecraftClient -> Quest.clientTicker.tick(minecraftClient.level, true));
     }
     
     @Override

--- a/common/src/main/java/hardcorequesting/common/proxies/CommonProxy.java
+++ b/common/src/main/java/hardcorequesting/common/proxies/CommonProxy.java
@@ -1,5 +1,6 @@
 package hardcorequesting.common.proxies;
 
+import hardcorequesting.common.HardcoreQuestingCore;
 import hardcorequesting.common.network.PacketContext;
 import hardcorequesting.common.quests.Quest;
 import hardcorequesting.common.quests.QuestTicker;
@@ -9,7 +10,8 @@ public class CommonProxy {
     public void initSounds() {}
     
     public void init() {
-        Quest.serverTicker = QuestTicker.initServerTicker();
+        Quest.serverTicker = new QuestTicker();
+        HardcoreQuestingCore.platform.registerOnServerTick(minecraftServer -> Quest.serverTicker.tick(minecraftServer.overworld(), false));
     }
     
     public boolean isClient() {

--- a/common/src/main/java/hardcorequesting/common/quests/QuestTicker.java
+++ b/common/src/main/java/hardcorequesting/common/quests/QuestTicker.java
@@ -1,28 +1,12 @@
 package hardcorequesting.common.quests;
 
-import hardcorequesting.common.HardcoreQuestingCore;
-import net.fabricmc.api.EnvType;
-import net.fabricmc.api.Environment;
 import net.minecraft.world.level.Level;
 
 public class QuestTicker {
     
     private long hours;
     
-    @Environment(EnvType.CLIENT)
-    public static QuestTicker initClientTicker() {
-        QuestTicker ticker = new QuestTicker();
-        HardcoreQuestingCore.platform.registerOnClientTick(minecraftClient -> ticker.tick(minecraftClient.level, true));
-        return ticker;
-    }
-    
-    public static QuestTicker initServerTicker() {
-        QuestTicker ticker = new QuestTicker();
-        HardcoreQuestingCore.platform.registerOnServerTick(minecraftServer -> ticker.tick(minecraftServer.overworld(), false));
-        return ticker;
-    }
-    
-    private QuestTicker() {}
+    public QuestTicker() {}
     
     public void tick(Level level, boolean isClient) {
         if (level != null && level.getGameTime() / 1000 != hours) {

